antsibull/build_ansible_commands.py

```diff
@@ -320,7 +320,7 @@ def build_single_command() -> int:
 
     build_single_impl(dependency_data)
 
-    deps_filename = os.path.join(app_ctx.extra['dest_dir'], app_ctx.extra['deps_file'])
+    deps_filename = os.path.join(app_ctx.extra['build_data_dir'], app_ctx.extra['deps_file'])
```

Comment:
This instance of using deps_filename is about writing so it should go to dest_dir.